docs/Using Fleet/MDM-macOS-setup.md

```diff
@@ -95,7 +204,23 @@ In the output you should see that package has a "signed" status.
 
 ### Step 3: upload the package to Fleet
 
-1. Upload the package to a storage location (ex. S3 or GitHub). During step 4, Fleet will retrieve the package from this storage location and host it for deloyment.
+In Fleet, you can upload the package using the Fleet UI or fleetctl command-line tool.
+
+The Fleet UI method is a good start if you're just getting familiar with Fleet.
+
+The fleetctl CLI method enables managing the package in a Git repository. This way you can enforce code review and benefit from Git's change history.
+
+Fleet UI:
+
+1. Head to the **Controls > macOS settings > macOS setup > Bootstrap package** page.
+
+2. Choose which team you want to add the bootstrap package to to by selecting the desired team in the teams dropdown in the upper left corner.
+
+3. Select **Upload** and choose your bootstrap package.
```

Comment:
Any required formats?

Reply:
Yes, the package has to be `.pkg`. This is clarified by the UI so I don't think we need to add it to the docs:
![Screenshot 2023-08-03 at 5 06 15 PM](https://github.com/fleetdm/fleet/assets/47070608/74e965ff-5569-41bb-bc42-bb3da3f07022)

User sees an error message if it's not a `.pkg`